mayastor/tests/nexus_multipath.rs

```diff
@@ -138,6 +140,51 @@ async fn nexus_multipath() {
         );
     }
 
+    let output_list = Command::new("nvme").args(&["list"]).output().unwrap();
```

Comment:
`nvme-cli` is not in our shell.nix right now, could you add it so this test won't fail on systems without `nvme`?

Reply:
hmm it is in the shell as well as nvmet-cli.

Reply:
It is indeed in `shell.nix` as part of `buildInputs`.

Reply:
Sorry, I looked in the mayastor pkg itself. :sweat: